features/bootstrap/JsonContext.php

```diff
@@ -15,6 +15,7 @@
 use Behatch\Context\JsonContext as BaseJsonContext;
 use Behatch\HttpCall\HttpCallResultPool;
 use Behatch\Json\Json;
+use PHPUnit\Framework\Assert;
```

Comment:
Do we really need to use PHPUnit classes in Behat? :rofl:

Reply:
1. We already do that in some contests
2. I don't want to bother reimplementing something that already is in PHPUnit 😅